widget/entry.go

```diff
@@ -474,6 +485,14 @@ func (e *Entry) Redo() {
 	e.Refresh()
 }
 
+func (e *Entry) Refresh() {
+	e.propertyLock.Lock()
+	e.minCache = nil
+	e.propertyLock.Unlock()
+
+	e.BaseWidget.Refresh()
+}
```

Comment:
I'm not saying that it need to be done now or should be done, but maybe it would make sense to have some kind of `MinSize` cache built in to the basewidget? I suppose it is something we might want to do for all widgets more or less (unless the computation is very cheap).

Reply:
I did propose the BaseWidget inclusion some time back but there was pushback for a reason I cannot recall.
Let's get it in here and see how useful it is and it could be pulled up without impact later.

Reply:
It might be worth exploring again. Either as part of the base widget or as some kind of ready cache-system that developers can add to their widgets (maybe using `async.Size` internally?). We should at least look at caching MinSize in more widgets :)